### ISSUE TITLE
Why use assembled genomes vs raw reads in example workflow 1

### ISSUE THREAD

**Issue by Lamm-a** [DRL/blobtools#107]
I would leave this question on the google group but the link t it is broken and I can't find it myself.

Why use an assembled genome bs raw reads in the example workflow on the [readme.io](https://blobtools.readme.io/docs/the-blobtools-workflows) as how can we be sure if some of the sequence in your assembly should not be there in first place?

**Comment by DRL** [comment]
Hi Lamm-a,

sorry, i don't understand the question.

please elaborate.

cheers,

dom

**Comment by Lamm-a** [comment]
Hi Dom,

I mean that why does Blobtools operate via detecting comtamination in an already assembled genome vs before assembly?

Would the down side of using an assembled genome not be that we can't be sure if some of the sequence in your assembly should not be there in first place as it might come from a contaminant?

**Comment by DRL** [comment]
`blobtools` just takes information about sequences (length, GC, sequence similarity, coverage, ...) and aggregates it for each sequence. 

> I mean that why does Blobtools operate via detecting comtamination in an already assembled genome vs before assembly?

before assembly there are only reads ... are you asking why are we not doing something with the reads? 

> Would the down side of using an assembled genome not be that we can't be sure if some of the sequence in your assembly should not be there in first place as it might come from a contaminant?

I don't understand what you think the alternative is.

**Comment by Lamm-a** [comment]
> before assembly there are only reads ... are you asking why are we not doing something with the reads?

Yes. Would it be possible to use mapped raw reads in a similar fashion?

Sorry if this is a ignorant question.
